test/runtests.jl

```diff
@@ -121,19 +122,62 @@ end
     @test length(testlogger.logs) == 2
 end
 
+@testset "DatetimeRotatingFileLogger" begin
+    mktempdir() do dir
+        drfl_sec = DatetimeRotatingFileLogger(dir, raw"\s\e\c-YYYY-mm-dd-HH-MM-SS.\l\o\g")
+        drfl_min = DatetimeRotatingFileLogger(dir, raw"\m\i\n-YYYY-mm-dd-HH-MM.\l\o\g")
+        sink = TeeLogger(drfl_sec, drfl_min)
+        with_logger(sink) do
+            while millisecond(now()) < 100 || millisecond(now()) > 200
+                sleep(0.001)
+            end
+            @info "first"
+            @info "second"
+            sleep(0.9)
```

Comment:
```suggestion
            sleep(0.9)  # just under 1 second so that we don't skip a second and get 3 files created
```

Should we worry about this being too fast and failing to make 2 files?

```suggestion
            sleep(0.95)  # just under 1 second so that we don't skip a second and get 3 files created
```

Reply:
I don't think so, since we've already slept until `millisecond(now())` is `>= 100`, and `sleep(0.9)` will always sleep for _at least_ 0.9 seconds.

Reply:
ok